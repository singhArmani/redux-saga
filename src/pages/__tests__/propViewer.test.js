import React from 'react';
import PropViewer from '../propViewer';

import renderer from 'react-test-renderer';

describe('PropViewer', () => {
  const testProps = {
    match: {
      url: 'testUrl',
      path: 'testPath',
      params: {}
    }
  };
  it('renders correctly', () => {
    const tree = renderer.create(<PropViewer match={testProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
