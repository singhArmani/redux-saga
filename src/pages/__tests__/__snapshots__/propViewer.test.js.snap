// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropViewer renders correctly 1`] = `
<h2>
   
   
</h2>
`;
